recipe/meta.yaml

```
@@ -1,36 +1,38 @@
 {% set name = "kombu" %}
-{% set version = "5.2.2" %}
-
+{% set version = "5.5.4" %}
+{% set u_tests_to_skip = "--ignore=gh_src/t/unit/transport/test_azurestoragequeues.py " %}
+{% set u_tests_to_skip = u_tests_to_skip + "--ignore=gh_src/t/unit/transport/test_gcpubsub.py " %}
+{% set u_tests_to_skip = u_tests_to_skip + "--ignore=gh_src/t/unit/transport/test_gcpubsub.py " %}
+{% set i_tests_to_skip = "--ignore=gh_src/t/integration/test_redis.py " %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kombu-{{ version }}.tar.gz
-  sha256: 0f5d0763fb916808f617b886697b2be28e6bc35026f08e679697fc814b48a608
-  patches:
-    - fix_test_txt.patch
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 886600168275ebeada93b888e831352fe578168342f0d1d5833d88ba0d847363
+  - url: https://github.com/celery/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 7793a288cc0316d06e66785a18182b8629a692a9fb2976b85c51cb466f95c008
+    folder: gh_src
 
 build:
   number: 0
-  skip: true  # [py<37]
+  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - patch     # [not win]
-    - m2-patch  # [win]
   host:
     - python
-    - pip
-    - setuptools
     - wheel
+    - pip
   run:
     - python
-    - importlib_metadata >=0.18  # [py<38]
-    - amqp >=5.0.6,<6.0.0
-    - vine
+    - amqp >=5.1.1,<6.0.0
+    - vine 5.1.0
+    - backports.zoneinfo >=0.2.1  # [py<39]
```

I _think_ for `backports.zoneinfo[tzdata]>=0.2.1; python_version<"3.9"` we also need `python-tzdata`
```suggestion
    - backports.zoneinfo >=0.2.1  # [py<39]
    - python-tzdata   # [py<39]
```